### ISSUE TITLE
missing String transform Type "Join"

### ISSUE THREAD

**Issue by haarchri** [crossplane-contrib/function-patch-and-transform#119]
we missing in p&t function String transform Type "Join"
was implemented in crossplane via https://github.com/crossplane/crossplane/pull/5148

**Comment by haarchri** [comment]
think after implementation we need to update the converter in crossplane cli too

**Comment by laseanb** [comment]
If this is true - you have saved me a bunch of troubleshooting headache!! I just tried using the "Join" feature but I got the error:
```sh
cannot compose resources: pipeline step "patch-and-transform-queue-policy" returned a fatal result: cannot get Function input: cannot get Function input %!T(MISSING) from %!T(MISSING), into, req: cannot unmarshal JSON from *structpb.Struct into *v1beta1.Resources: json: cannot unmarshal Go value of type v1beta1.StringTransform: unknown name "join"
```

**Comment by negz** [comment]
What we really need to do is https://github.com/crossplane/crossplane/issues/4746 to eliminate this entire class of issues. 🙂

**Comment by gschei** [comment]
@haarchri this was implemented with #133 and release with v0.7.0, issue can be closed
